setup.cfg

```diff
@@ -77,6 +77,8 @@ all =
     h5py
     nni
     optuna
+    onnx>=1.13.0
+    onnxruntime
```

Comment:
Hi @binliunls ,

Please add below things in this postion:
https://github.com/Project-MONAI/MONAI/blob/dev/setup.cfg#L135
```
nni =
    nni
optuna =
    optuna
onnx =
    onnx>=1.13.0
    onnxruntime
```